charts/am-org-role-mapping-service/Chart.yaml

```diff
@@ -3,16 +3,16 @@ appVersion: "1.0"
 description: A Helm chart for AM Organisation Role Mapping Service
 name: am-org-role-mapping-service
 home: https://github.com/hmcts/am-org-role-mapping-service
-version: 0.0.70
+version: 0.0.71
 maintainers:
   - name: Access Management Team
 dependencies:
   - name: java
-    version: 5.2.1
-    repository: 'https://hmctspublic.azurecr.io/helm/v1/repo/'
+    version: 5.3.0
+    repository: 'oci://hmctspublic.azurecr.io/helm'
   - name: servicebus
-    version: 1.0.7
-    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 1.1.0
+    repository: 'oci://hmctspublic.azurecr.io/helm'
     condition: orm.servicebus.enabled
   - name: am-role-assignment-service
     version: 0.0.80
```

Comment:
This PR should be completed after RAS and JBS are complete so the two values for RAS and JBS can also be updated to use OCI.

Reply:
So the following additional references also need switching over to the **OCI** url and the new version numbers:
* `am-role-assignment-service` needs updating to reflect COT-390 hmcts/am-role-assignment-service#2474 (**0.0.81**)
* `am-judicial-booking-service` needs updating to reflect COT-392 hmcts/am-judicial-booking-service#1254 (**0.0.64**)

Reply:
(The above is based on the assumption that the OCI change affects all our chart files in HMCTS)

Reply:
bumped RAS and JBS to oci versions

Reply:
Resolved in 9f78766d914f60481256651d4a367f4a6b9e1b07.